packages/bootstrap-4/src/BaseInputTemplate/BaseInputTemplate.tsx

```diff
@@ -62,6 +63,7 @@ export default function BaseInputTemplate<
         onChange={_onChange}
         onBlur={_onBlur}
         onFocus={_onFocus}
+        aria-describedby={ariaDescribedByIds<T>(id)}
       />
       {children}
       {schema.examples ? (
```

Comment:
Consider adding an ID generator function for `examples` below. It would probably make sense to add that ID to `aria-describedby`

Reply:
Hmmm, good point... Although that would make the description pretty long and involved.

Reply:
I like the choice to make it optional via parameter